### DISCUSSION TITLE
What happened to my points?!?

### DISCUSSION THREAD

**Discussion by CodeByAidan** [codewars/codewars.com#2819] [Q&A] [answered]
I completed my first ever 2 kyu's and submitted 3 of them that I worked on for over a week and then i got up from 500 kata to 1000 kata. 

14 hours later, I lost my solutions to those kata problems, and now I'm a 5 kyu and lost over 610 points?
What is going on? No notice or anything...? 

My user is livxy, I also post my solutions on my github?

**Comment by Blind4Basics** [comment]
Hello,

Are you on CW discord, by any chance? If so, could you poke at me in DM (`B1³B`)

> I also post my solutions on my github?

don't do that, plz...

**Reply by CodeByAidan** [reply]
will do!

**Comment by Blind4Basics** [comment, marked as answer]
.
